horse/users.py

```diff
@@ -61,3 +67,15 @@ def post(self, user_pk):
 
 
 users_api.add_resource(UserFollow, '/users/<string:user_pk>/follow')
+
+
+class UserLikesMovie(Resource):
+    def post(self, user_pk, movie_pk):
+        user = get_user_by_pk(user_pk)
+        movie = get_movie_by_pk(movie_pk)
+        user.add_to_liked_movies(movie)
+
+
+users_api.add_resource(
+    UserLikesMovie,
+    '/users/<string:user_pk>/movies/<string:movie_pk>/like')
```

Comment:
I think it should be similar to following users, because right now it suggest that movies exist only in context of given user, maybe `/users/<string:user_pk>/like_movie` ?